node.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:stretch-slim
```

Comment:
Why not use `FROM node`? Also there is already a discussion about including node in the moodle-apache-php image

Reply:
```suggestion
FROM node
```